eng/pipelines/runtime-staging.yml

```diff
@@ -177,8 +177,8 @@ jobs:
       extraStepsTemplate: /eng/pipelines/libraries/helix.yml
       extraStepsParameters:
         creator: dotnet-bot
-        interpreter: true
```

Comment:
Do we no longer need interpreter true for `Full AOT` on devices?

Reply:
No, full-aot doesn't need interp.  I'm not sure why the extra param was there to begin with